### ISSUE TITLE
The perfect seed.

### ISSUE THREAD

**Issue by michael** [michael/hub#72]
For developing a proper UI of the composer we need some good example data. So a great basis to work with would be the current set of eLife documents.

Can we create a dynamic seeding mechanism for dev that would just pull in all the eLife docs from:

http://elife-converter.herokuapp.com/documents

and for each document:

http://elife-converter.herokuapp.com/documents/00048

?

**Comment by Integral** [comment]
Hi! Did you mean parsing that data and save it in our database?

**Comment by michael** [comment]
Yep. Should be fairly simple since the eLife format already conforms to the substance doc format. So actually no parsing involved here ;)

**Comment by Integral** [comment]
Sorry, missed some changes. Could you point me to db API?
So we need something like button "load test data" to get all data from lens API? 
And sorry - I'm not pretty familiar with document model. Is it true that `type` object is something like schema for `objects`? So http://elife-converter.herokuapp.com/documents is part of document model too?

**Comment by Integral** [comment]
So I see many properties in lens schema. Do we need to save them? Or we just need to use our schema which is described in `schema.json`?

**Comment by michael** [comment]
Don't worry about it for now.. we changed so much, it just takes a while until the code is consistent again. Will keep you posted. :)

**Comment by Integral** [comment]
Thx. Sorry.

**Comment by michael** [comment]
I'm currently working on a big redesign...  you could definitely help by creating design sketches. How much time will you be able to spend working on Substance in the coming 6 weeks ? Just for rough task planning. We'd like to get a solid release ready before end of summer.. so there's definitely a lot of stuff that we could need help with.

**Comment by michael** [comment]
first mockup of the new vertical split layout..

![screen shot 2013-06-25 at 4 24 56 pm](https://f.cloud.github.com/assets/2931/703077/1b9a7d62-dda3-11e2-8293-5d0328791428.png)

**Comment by Integral** [comment]
Looks fascinatingly. Well, I think that I could spend about 6 hours per
week. I can help with sketches/css/jade/local javascript.

**Comment by Integral** [comment]
Or with backbone if you have plans to use views like with lenses.

**Comment by michael** [comment]
You're more than welcome. :)  I'll think about suitable tasks that can be done independently. Will assign issues to you, and you can choose which ones you like to take. Just gimme some more time :)

**Comment by Integral** [comment]
Cool! I'll be waiting. But I'm sorry if I'll ask you a lot of questions :)
And sorry for bumping this issue.
Thanks!

**Comment by michael** [comment]
I hope we have working version of the stack by the end of the week. I'll give you instructions how to install. It's hard to keep in sync the documentation since we're changing so much.
